docs/modules/parameters/pages/action-scope.adoc

```diff
@@ -0,0 +1,82 @@
+= Action scope
```

Comment:
This action is used in one step only, so there is no reason to extract it to a separate page, please join this doc with the step definition

Reply:
refactored